dev-infrastructure/docs/development-setup.md

```diff
@@ -353,8 +355,44 @@ Then register it with the Maestro Server
     ```
     az network vnet subnet update -g <resource-group> -n <subnet-name> --vnet-name <vnet-name> --network-security-group <nsg-name>
     ```
+  - Generate a random alphanumeric string used as a suffix for the User-Assigned Managed Identities of the operators of the cluster
+    > NOTE: The random suffix used has to be different for each cluster to be created
+    ```
+    export OPERATORS_UAMIS_SUFFIX=$(openssl rand -hex 3)
+    ```
+  - Create the User-Assigned Managed Identities for the Control Plane operators. This assumes OCP 4.17 based will be created.
```

Comment:
These identities can be created in any resource group is that correct? Similar to vnet+nsg?

Reply:
The resource group must be within the same subscription as the cluster's subscription